README.md

```diff
@@ -80,6 +91,8 @@ Thus plugin use the new [miio](https://github.com/aholstenson/miio) version 0.15
 | `pause` | false | when set to true, HomeKit shows an additional switch for "pause" - switch is on, when pause is possible |
 | `dock` | false |  when set to true, HomeKit shows an occupancy sensor, if robot is in the charging dock |
 | `waterBox` | false | when set to true, HomeKit shows an additional slider to control the amount of water released by the robot (only selected models like S5-Max). Currently in a beta state. |
+| `cleanword` | cleaning | used for autonaming the Roomselectors |
+| `rooms` | false | Array of ID / Name for a single Room. If set you have another switch for cleaning only this room |
```

Comment:
It would be nice to have an explanation on how to get those IDs or we'll get tons of issues asking for it 😅